BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m

```diff
@@ -20,24 +20,38 @@ + (UIImage *)box_iconForItem:(BOXItem *)item
 
     UIImage *icon = nil;
 
-    if (item.isFolder) {
-        BOXFolder *folder = (BOXFolder *)item;
-        if (folder.isExternallyOwned == BOXAPIBooleanYES) {
-            icon = [UIImage imageNamed:@"icon-folder-external" inBundle:browseResourceBundle compatibleWithTraitCollection:nil];
-        } else if (folder.hasCollaborations == BOXAPIBooleanYES) {
-            icon = [UIImage imageNamed:@"icon-folder-shared" inBundle:browseResourceBundle compatibleWithTraitCollection:nil];
-        } else {
-            icon = [UIImage imageNamed:@"icon-folder" inBundle:browseResourceBundle compatibleWithTraitCollection:nil];
+    @synchronized (browseResourceBundle) {
```

Comment:
The only actual change is adding this @synchronized block. Otherwise, the rest of it is formatting changes only (newlines for long method names).